### PR TITLE
[dom] CDO 1.9.5 with 19.03

### PR DESCRIPTION
**Pull request by kraushm**

### CODE REVIEW

**Comment by vkarak** [comment]
@kraushm Could you resolve the conflicts?

File: jenkins-builds/7.0.UP00-19.03-gpu [hunk 1]
```diff
@@ -29,4 +29,4 @@
  papi-wrap-1.0-CrayGNU-19.03.eb       --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP00/gpu/easybuild/tools
  papi-wrap-1.0-CrayCCE-19.03.eb                            --installpath=$APPS/UES/jenkins/7.0.UP00/gpu/easybuild/tools
  papi-wrap-1.0-CrayIntel-19.03.eb                          --installpath=$APPS/UES/jenkins/7.0.UP00/gpu/easybuild/tools
- likwid-4.3.3-perf_event.eb           --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP00/gpu/easybuild/tools
+ likwid-4.3.3-perf_event.eb           --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP00/gpu/easybuild/tools
```
Comment (vkarak) [inline jenkins-builds/7.0.UP00-19.03-gpu]: Do a checkout from the master for these files. This PR should not touch them.

Comment (lucamar) [reply jenkins-builds/7.0.UP00-19.03-gpu]: @vkarak I have removed the production lists from this pull request

**Comment by lucamar** [comment]
@vkarak Please accept the changes to merge this pull request

**Pull request merged**